docs/learn/learn-comparisons.md

```diff
@@ -1,63 +1,212 @@
+| **Security Model**           | Relies on cryptographic validity proofs, ensuring high security and no reliance on game-theoretic assumptions.                                                                                                                                                 | Relies on economic incentives and a challenge period to catch fraud. Optimistic assumption makes it less secure than ZK Rollups.                                             | "Cynical" model, where every transaction is checked by a subset of validators, with escalation and slashing in case of disputes.                                                                                                                                                                                                                    |
+| **Scalability**              | Limited scalability as they are a single state machine and are only as scalable as the prover machine and computation requirement. Many zk-rollups disable cryptographic precompiles on mainnet as a result of the immense computational requirement for them. | High, with parallelization, but limited by gas costs on L1 chains like Ethereum.                                                                                             | Inherently scalable through native execution sharding and parachains operating in parallel. [Pipelining](./learn-async-backing.md) and [core scheduling](./learn-agile-coretime.md) increase throughput and scalability for the single rollup. Execution sharding is enabled by multiple virtual cores using [coretime](./learn-agile-coretime.md). |
+| **Decentralization**         | Centralization risks for sequencers and zk proof computation.                                                                                                                                                                                                  | Sequencer centralization risks; fewer validators can lead to potential censorship.                                                                                           | Collator decentralization is critical to ensure parachains remain trustless and independent.                                                                                                                                                                                                                                                        |
```

Comment:
Sequencer centralization risks are common problems to all rollups polkadot or not